release-notes/con-relnotes-known-issues.adoc

```diff
@@ -36,3 +36,7 @@ In the new backend system, automatically getting the IDs of all dynamically inst
 *Workaround*: You can manually add a list of dynamic plugin IDs supporting RBAC in the {product-short} application configuration file.
 
 --
+
+Issues with OpenShift Container Platform 4.15.3 deployment::
+Deploy your {product-short}} instance on OpenShift Container Platform 4.15.2 or earlier versions to avoid known issues reported with OpenShift Container Platform 4.15.3 deployment. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-31080[OCPBUGS-31080]. 
```

Comment:
```suggestion
Deploy your {product-short} instance on OpenShift Container Platform 4.15.2 or earlier versions to avoid known issues reported with OpenShift Container Platform 4.15.3 deployment. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-31080[OCPBUGS-31080]. 
```